change/@fluentui-web-components-01f6ae19-5f09-445f-b158-5808d3fb0b7a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29009&commitId=93d4df6d1b12b9b7a965a1f74d3b314a364acc7c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.hover.chromium.png|104| Changed|
|Switch.Checked - RTL.normal.chromium.png|97| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>